testsuite/oiiotool-color/run.py

```diff
@@ -62,6 +62,10 @@
                      + "--ccmatrix 0.805,0.506,-0.311,0,-0.311,0.805,0.506,0,0.506,-0.311,0.805,0,0,0,0,1 "
                      + "-d uint8 -o tahoe-ccmatrix.tif")
 
+# test test we can read an OCIOv2 config -- when building against OCIOv1,
```

Comment:
1. There is a  typo 'test test' -> 'test'
2. Reading a v2 config using OCIOv1 must fail so, the test description and the test result message are ambiguous !

Reply:
will fix, thanks